parts/k8s/kubernetescustomscript.sh

```diff
@@ -560,10 +593,12 @@ if [ -f $CUSTOM_SEARCH_DOMAIN_SCRIPT ]; then
     $CUSTOM_SEARCH_DOMAIN_SCRIPT > /opt/azure/containers/setup-custom-search-domain.log 2>&1 || exit $ERR_CUSTOM_SEARCH_DOMAINS_FAIL
 fi
 
+holdWALinuxAgent
 installDeps
+ensureRpc
```

Comment:
I don't see the `ensureRpc` function defined

Reply:
typo, fixed.